bundler/benchmarks/bm_bundle_install_rails.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'support/benchmark_bundler.rb'
+require 'bundler/cli'
+
+
+Dir.mktmpdir do |dir|
+  Dir.chdir(dir) do
+    File.write('Gemfile', <<-GEMFILE)
+      source 'https://rubygems.org'
+      gem 'rails', '~> 5.0.0'
+      gem 'nokogiri', '1.6.7.2'
+    GEMFILE
+
+    system("bundle install --quiet")
```

Comment:
The real issue is that network connection times and disk I/O dominate these operations, and they're going to vary a lot